tokio/src/signal/registry.rs

```diff
@@ -175,6 +175,7 @@ where
     GLOBALS.as_ref()
 }
 
+/*
```

Comment:
Any reason this was commented out, or left over debug code?

Reply:
Oh, no good catch.